### ISSUE TITLE
Wrong upgrade url

### ISSUE THREAD

**Issue by jwald1** [rails/solid_queue#273]
Bundle printed the following after upgrading to 0.4.0
```bash
Post-install message from solid_queue:
Upgrading to Solid Queue 0.4.x? There are some breaking changes about how Solid Queue is started. Check
https://github.com/rails/solid_cache/blob/main/UPGRADING.md for upgrade instructions.
```
 The URL seems to be incorrect.

**Comment by rosa** [comment]
Oops, indeed 🤦‍♀️ I'll fix it.

**Comment by rosa** [comment]
Should be fixed in version 0.4.1. Thanks for letting me know!
